src/main/java/mil/dds/anet/database/PositionDao.java

```diff
@@ -397,6 +485,13 @@ public int deletePositionAssociation(String positionUuidA, String positionUuidB)
         .thenApply(l -> PersonPositionHistory.getDerivedHistory(l));
   }
 
+  public CompletableFuture<List<PersonPositionHistory>> getAllPositionHistory(
```

Comment:
This method is only (indirectly) called by the tests, so really belongs somewhere under `src/test/java`.

Reply:
This method is moved under `src/test/java`.